src/ServicePulse.Host/angular/app/js/.eslintrc.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
     "env": {
         "browser": true,
-        "es6": false,
+        "es6": true,
```

Comment:
If I remember right from how you explained it: This is OK because we don't support IE anymore. Correct?

Reply:
Correct